packages/http-server/src/serializers.ts

```diff
@@ -18,4 +19,17 @@ export default [
     },
     serializer: (data: unknown) => (typeof data === 'string' ? data : xmlSerializer.parse(data)),
   },
+  {
+    regex: /text\/plain/,
+    serializer: (data: unknown) => {
+      if (typeof data === 'string') return data;
```

Comment:
needs them sweet `{}`
also... I know we talked about, but maybe:
`if (condition) {} else {}` 
I strongly believe, that this way we show our reasoning better. But this is just a non-blocking suggestion :)

Reply:
Fine with the {} suggestion but please no unnecessary conditionals: https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
Ha! Martin Fowler trick ;> yeah, I don't agree with this rule personally. So, yeah, as I said, just a suggestion and personal preference.